src/M6Web/Component/RedisMock/RedisMockFactory.php

```diff
@@ -322,6 +325,12 @@ protected function getMethodSignature(\ReflectionMethod $method)
                 } else {
                     $signature .= var_export($parameter->getDefaultValue(), true);
                 }
+            } else if ($parameter->isOptional() && !$variadic) {
+                $signature .= ' = null';
+            }
```

Comment:
Which cases do you intend to address? 🤔 
If the parameter is optional, why it's default value should be `null`? What about a signature like this: `function foo($bar = true) {}`